packages/usa-nav/src/test/navigation.spec.js

```diff
@@ -163,9 +180,10 @@ describe("navigation toggle", () => {
 
   it("collapses dropdowns when focus leaves nav", () => {
     menuButton.click();
-    navLink.click();
-    EVENTS.focusOut(navLink);
-    assert.strictEqual(isVisible(nav), false);
+    accordionButton.click();
+    assert.strictEqual(accordionButton.getAttribute("aria-expanded"), "true");
+    EVENTS.focusOut(navPrimary);
+    assert.strictEqual(accordionButton.getAttribute("aria-expanded"), "false");
```

Comment:
## Valid test

This test properly:

1. Opens navigation
2. Opens navigation dropdown/accordion
3. Confirms dropdown is expanded
4. Focuses out of the `usa-nav__primary` section, which properly emulates current focus change states
5. Confirms the dropdown is now closed